services/softlayer_virtual_guest_test.go

```diff
@@ -2127,4 +2127,41 @@ var _ = Describe("SoftLayer_Virtual_Guest_Service", func() {
 			})
 		})
 	})
+
+	Context("#GetLocalDiskFlag", func() {
+		BeforeEach(func() {
```

Comment:
Please add some error path tests. That is test to check when `false` is returned as well as `false` with error